moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1170,6 +1170,14 @@ bool MoveItConfigData::outputJointLimitsYAML(const std::string& file_path)
 
     const moveit::core::VariableBounds& b = joint->getVariableBounds()[0];
 
+    // Output property
+    emitter << YAML::Key << "has_position_limits";
+    emitter << YAML::Value << "false";
```

Comment:
why `false`? I'm not even sure what happens when you set that while the URDF actually has a position limit